src/Symfony/Component/ErrorHandler/Tests/Fixtures/ReturnTypeParent.php

```diff
@@ -199,13 +199,6 @@ public function mixed()
     {
     }
 
-    /**
-     * @return mixed|null
-     */
-    public function nullableMixed()
-    {
-    }
-
```

Comment:
This fixture is still valid. We still need to be able to work with such code even if we decide that this piece code does not match our code style.

Reply:
why, for what? There is `mixed()` method.

Reply:
Right, but it does represent a different case, doesn't it?

Reply:
I do not think so, that is why I am asking to verify :)

Reply:
Okay, well, it does. Keep the fixture, please.